### DISCUSSION TITLE
en/latest/CommunityDiscussion/%E7%94%A8%E6%88%B7%E4%BA%A4%E6%B5%81/

### DISCUSSION THREAD

**Discussion by giscus** [zdsjjtTLG/TrackIt#6] [General]
# en/latest/CommunityDiscussion/%E7%94%A8%E6%88%B7%E4%BA%A4%E6%B5%81/

在python中轻松进行路径匹配

https://gotrackit.readthedocs.io/en/latest/CommunityDiscussion/%E7%94%A8%E6%88%B7%E4%BA%A4%E6%B5%81/

**Comment by zdsjjtTLG** [comment]
# 这里是评论区
为了避免重复提问，请大家在手册下方的评论区提问，历史问题都会保留在评论区，方便大家快速查阅，评论区的内容也会自动同步到项目主页的[讨论区](https://github.com/zdsjjtTLG/TrackIt/discussions)

**Comment by zdsjjtTLG** [comment]
使用GitHub账号登陆后即可进行评论和提问
